modules/ROOT/pages/configure-tls-context-rtf-lb.adoc

```diff
@@ -0,0 +1,54 @@
+= Configure a TLS Context for Anypoint Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. When you configure inbound traffic for Runtime Fabric, you can configure the TLS-server by importing a Privacy-Enhance Mail (PEM) or Java Keystore (JKS), or by importing a TLS context from secrets manager (advanced use case).
+
+Before you can import the TLS Context from secrets manager, you need to add the TLS Context. 
+
+== Add the TLS Context to a Secret Group
+
+Adding a TLS context to your secret group involves supplying a name, target, security information, version, keystore and, optionally, truststore. 
+
+. In the *Secret Groups* list view, select the secret group to which to add a TLS context. 
+. Select *TLS Context* in the menu on the left, and click *Add TLS Context*. 
+. In the Create TLS Context screen, add the required information:
++
+* *Name* +
+Enter a name for your TLS context. 
+* *Target* +
+Select *Anypoint Security* to validate the SSL handshake for Runtime Fabric. 
+* *TLS Version* +
+By default, TLS 1.2 and TLS 1.3 are selected. +
+** TLS 1.1
+** TLS 1.2
+** TLS 1.3  
+* *Keystore* +
+From the drop-down list, select the keystore to store the TLS context. +
+You can select keystores and truststores that are either PEM or non-PEM types. Supported types include: +
+*** JKS
+*** PKCS #12
+*** JCEKS 
+* *Truststore* +
+Optionally select a truststore to which to add the TLS context if you are using a truststore to store certificates trusted by the client. +
+* *Expiration Date* +
+Optionally select the expiration date for the certificate.
+. Optionally xref:enable-client-authentication.adoc[enable client authentication]. 
+. Optionally xref:cipher-suites.adoc[select ciphers]. 
+. Click *Save*.
+
+[[import_tls_rtf]]
+== Import a TLS Context to Runtime Fabric
+
+. In *Runtime Manager > Runtime Fabrics > Inbound Traffic*, select *Import from Secrets Manager*. 
+. Select the environment the secret group that contains the TLS context is associated with.
+. Select the secret group where the TLS context is configured.
```

Comment:
What does this mean? the customer needs to first select the environment and then select the secret group?

Reply:
Yes, and this is clear to the user.